src/styles/variables.scss

```diff
@@ -0,0 +1,20 @@
+$card-title-size: 12px;
+$card-title-weight: 500;
+$card-font-color: #060b35;
+$card-width: 200px;
+$img-width: 160px;
+$img-height: 134;
+$card-horiz-margin: 16px;
+$product-code-size: 10px;
+$product-code-color: #616070;
+$cta-color: #00acdc;
+$cta-hover-color: #00acdc;
```

Comment:
The `$cta-hover-color` variable has the same value as `$cta-color`. If this is intentional and you want to use the same color for both states, consider using only one variable to reduce redundancy. If the hover color is supposed to be different, update the value accordingly.